### DISCUSSION TITLE
How about join the efforts.

### DISCUSSION THREAD

**Discussion by chiddekel** [AvaloniaUI/Avalonia#6829] [General]
How aboute join efforts with software developer of project [Uno platform](https://github.com/unoplatform)?

**Comment by robloo** [comment]
I slipped this idea into a discussion some time ago to see if there would be any traction. The response from Francois (nventive/Uno CEO) was as follows:

 > As for the unrelated topic of Uno sitting on top of Avalonia, it doesn't make sense to us. First, we already have a Skia backend if/when needed. Second, we took an explicit direction of aligning with Microsoft's ecosystem so third party components can easily be reused.

https://github.com/unoplatform/uno/discussions/6097#discussioncomment-967738 (See point number 3)

While both are multi-platform UI layers, the direction of both technologies is quite different. Architecture is also quite different as well (except for Uno's Skia backend). Bottom line Avalonia does it's own thing and that allows it to evolve with new ideas at a much faster pace (at the expense of existing code/third-party controls). Uno is following Microsoft with all the baggage that entails. This was discussed for Avalonia here: https://github.com/AvaloniaUI/Avalonia/discussions/5596.

Personally, I would love to see both WPF and WinUI controls/API sitting on top of Avalonia in the future. It would be great to have everyone using the same underlying multi-platform UI layer. This would open a ton of doors for bringing over existing apps and commercially would be quite useful. WPF is probably the most realistic right now to start with. However, even that task is quite daunting. The Wine people have a lot of experience using that code and getting it to work multi-platform too.

As a side note, I think this needs to be moved to Discussions. It isn't an issue.
